curp/src/server/raw_curp/mod.rs

```diff
@@ -1342,6 +1395,10 @@ impl<C: Command, RC: RoleChange> RawCurp<C, RC> {
             self.ctx.role_change.on_calibrate();
         }
         st.term = term;
+        if st.role == Role::Leader {
+            // a leader fallback into the follower
+            metrics::get().leader_changes.add(1, &[]);
+        }
```

Comment:
We can merge this block of code with lines 1393 ~ 1396

Reply:
Done